test/account.access_key.test.js

```diff
@@ -69,10 +69,17 @@ test('view account details after adding access keys', async() => {
 
 test('loading account after adding a full key', async() => {
     const keyPair = nearlib.utils.KeyPair.fromRandom('ed25519');
-    await workingAccount.addKey(keyPair.getPublicKey());
+    // wallet calls this with an empty string for contract id and method
+    await workingAccount.addKey(keyPair.getPublicKey(), '', '');
```

Comment:
Why do we do this when we've already made contractId optional?

Reply:
This is how wallet calls the API. It's easy to make that mistake when dealing with UX components that may do a '' instead of null, so we should make sure it works on nearlib level